salem/tests/test_gis.py

```diff
@@ -1021,7 +1021,7 @@ def test_pyproj_trafo(self):
         y = np.random.randn(int(1e6)) * 60
 
         for i in np.arange(3):
-            xx, yy = pyproj.transform(wgs84, wgs84, x, y)
```

Comment:
This could be replace by `gis.transform_proj`, avoiding an external call to a private func

Reply:
Wouldn't that make this block identical to the one directly following it?

Reply:
:sweat_smile: it depends on what I was testing here - I let you judge of that, it's not too important

Reply:
I just removed those parts, since the two functions should behave identical anyway.